includes/templates/bootstrap/css/stylesheet_zca_colors.php

```diff
@@ -79,12 +83,12 @@
 ?>
 .btn {
     color: <?php echo ZCA_BUTTON_TEXT_COLOR; ?>;
-    background-color: <?php echo ZCA_BUTTON_COLOR; ?>;
+    background-color: <?php echo ZCA_BUTTON_BACKGROUND_COLOR; ?>;
     border-color: <?php echo ZCA_BUTTON_BORDER_COLOR; ?>;
 }
 .btn:hover {
     color: <?php echo ZCA_BUTTON_TEXT_COLOR_HOVER; ?>;
-    background-color: <?php echo ZCA_BUTTON_COLOR_HOVER; ?>;
+    background-color: <?php echo ZCA_BUTTON_BACKGROUND_COLOR_HOVER; ?>;
```

Comment:
I'm not a big fan of this renaming.  I know it's more consistent, but if a site has provided additional styling using those colors' constants the site will be met with a PHP Warning (at best).